staging/src/kubevirt.io/client-go/README.md

```diff
@@ -2,6 +2,10 @@
 
 Go clients for talking to [KubeVirt](https://github.com/kubevirt/kubevirt).
 
+## How to use it
+please refer to some of [examples here](examples/Readme.md)
```

Comment:
@kunalkushwaha the link is broken, README.md is all uppercase. Do you mind fixing it with a new PR?

Reply:
I will do it now.

Reply:
@slintes I added https://github.com/kubevirt/kubevirt/pull/2575 to fix this. Sorry for typo.

Reply:
No need for sorry. We also missed it in the reviews. Thx for the quick fix and the example at all! :+1: